.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
```

Comment:
Shouldn't we be assigning bugs ourselves instead of letting the author do it?

Reply:
We'll recategorise them if they aren't "real" bugs.